### ISSUE TITLE
Rule: document-title changes

### ISSUE THREAD

**Issue by jkodu** [dequelabs/axe-core#1146]
The following test cases should yield desired results

1) Should fail: Empty `title`.
```html
<html>
  <title> <!-- this page has an empty title --> </title>
</html>
```
2) Should fail: Empty first `title`.
```html
<html>
  <head>
    <title> <!-- this page has an empty first title --> </title>
  </head>
  <body>
    <title>Title of the page.</title>
  </body>
</html>
```

3) Should be inapplicable: Not applicable to `svg` element.
```html
<svg>
  <title>This is an SVG</title>
</svg>
```

**Comment by jkodu** [comment]
Content with in comment blocks of a `title` tag is returned as `document.title` - `<!-- this page has an empty first title -->`

Issues resolved by updating test cases in wai-tools
